src/TextUI/TestRunner.php

```diff
@@ -470,34 +470,30 @@ public function doRun(Test $suite, array $arguments = [], array $warnings = [],
             if (isset($arguments['configuration'])) {
                 $filterConfiguration = $arguments['configuration']->getFilterConfiguration();
```

Comment:
Running 

```
$ git grep getFilterConfiguration
```

yields

```
src/TextUI/TestRunner.php:471:                $filterConfiguration = $arguments['configuration']->getFilterConfiguration();
src/Util/Configuration.php:138:    public function getFilterConfiguration(): array
tests/unit/Util/ConfigurationTest.php:196:            $this->configuration->getFilterConfiguration()
tests/unit/Util/ConfigurationTest.php:554:        $filter = $emptyConfiguration->getFilterConfiguration();
tests/unit/Util/ConfigurationTest.php:617:            $expectedConfiguration->getFilterConfiguration(),
tests/unit/Util/ConfigurationTest.php:618:            $actualConfiguration->getFilterConfiguration()
```

The only class that declares this method is `final` and does not implement an interface:

https://github.com/sebastianbergmann/phpunit/blob/0d3b3170a38f876773df670bdf612ea33875f9a2/src/Util/Configuration.php#L24

The method `getFilterConfiguration()` returns an array where the value for key `whitelist` is never empty:

https://github.com/sebastianbergmann/phpunit/blob/0d3b3170a38f876773df670bdf612ea33875f9a2/src/Util/Configuration.php#L138-L199